frontend/src/routes/Embed.tsx

```diff
@@ -53,7 +53,7 @@ export const EmbedVideoRoute = makeRoute(url => {
     const queryRef = loadQuery<EmbedQuery>(query, { id: eventId });
 
     return {
-        render: () => <Suspense fallback={
+        render: () => <ErrorBoundary><Suspense fallback={
```

Comment:
As above: I don't like this formatting, please add a newline before `<Suspense>`. I'm also pretty sure we don't have this style anywhere in the codebase already.